src/main/java/database/helper/Settings.java

```diff
@@ -14,19 +14,24 @@
 import org.slf4j.LoggerFactory;
 
 public class Settings {
+    static final String DATABASE_STORAGE_PATH_PROPERTY_NAME = "database.storage.path";
+    static final String DEFAULT_DATABASE_STORAGE_PATH = "database";
+    static final int DEFAULT_LIMIT_VALUE = 100;
+    private static final String LIMIT_PROPERTY_NAME = "limit";
+    private static final Logger LOG = LoggerFactory.getLogger(Settings.class);
     private String propertyFileName;
```

Comment:
Please use the right order of the fields declaration:
- private final variables
- private non-final variables
- non-private variables
- static variables